compiler/src/dotty/tools/dotc/core/SymDenotations.scala

```diff
@@ -373,6 +374,20 @@ object SymDenotations {
       case Nil => Nil
     }
 
+    /** If this is a method, the parameter symbols, by section.
+     *  Both type and value parameters are included. Empty sections are skipped.
+     */
+    final def paramss: List[List[Symbol]] = myParamss
```

Comment:
This will need to be exposed from tasty-reflect to give access to parameter annotations. @nicolasstucki do you mind doing that ?

Reply:
I can do that once this is merged.